### PR TITLE
#489 enable UDP queries for MyLA cron job

### PR DESCRIPTION
**Pull request by zqian**
Test Plan:
- run `cp scripts/data_validation/env_validation_sample.hjson scripts/data_validation/env_validation.hjson`
- fill in `DATA_WAREHOUSE_COURSE_IDS` and `DATA_WAREHOUSE_SHORT_COURSE_IDS` with values for test courses 
- You will also need to update the DATA_WAREHOUSE inside mylasecrets/env.hjson to point to UDP context_store, and then run `docker exec -it student_dashboard /bin/bash -c "python manage.py migrate django_cron && python manage.py runcrons --force"` to start the cron job.
- run the validation script to compare UDW and UDP results: `docker exec -it student_dashboard /bin/bash -c "python scripts/data_validation/validate_udw_vs_udp.py"`

### CODE REVIEW

**Comment by jonespm** [comment]
@zqian The initial comment mentioned these were renamed, but I don't see these moves. And the validation can't find this file?

* moved cron.hjson to cron_udw.hjson;
* moved cron_udp.hjson to cron.hjson;

It looks like that switched back in this commit today? bf98cb781901f53e5a8ca8ad14e41829d7b82dfc

Should this comment be updated and the validation also updated?

File: scripts/data_validation/validate_udw_vs_udp.py [hunk 1]
```diff
@@ -71,16 +81,17 @@ def main():
...
-    print(os.path.abspath(os.path.abspath(os.pardir)))
+    logger.debug(dir_path)
+    logger.debug(os.path.abspath(os.path.abspath(os.pardir)))
     ENV_UDW = get_env_file('/secrets/env.hjson', 'hjson')
     ENV_UDP = get_env_file('/secrets/env_udp.hjson', 'hjson')
-    ENV_CRON_UDW = get_env_file('/secrets/cron.hjson', 'hjson')
-    ENV_CRON_UDP = get_env_file('/secrets/cron_udp.hjson', 'hjson')
+
+    # Use the config files in this project
+    ENV_CRON_UDW = get_env_file('/code/config/cron_udw.hjson', 'hjson')
```
Comment (jonespm) [inline scripts/data_validation/validate_udw_vs_udp.py:90]: This file no longer exists in the latest code.

Comment (zqian) [reply scripts/data_validation/validate_udw_vs_udp.py:90]: yes. I need to update this accordingly. Doing it now.

Comment (zqian) [reply scripts/data_validation/validate_udw_vs_udp.py]: @jonespm you can check [9739a47](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1392/commits/9739a47bd43f312781d84082976b20ec8ec5aa08) tomorrow.

**Comment by jonespm** [comment]
I ran the validation and everything looks fine except the course `start_at` and `conclude_at` for course don't match. It seems like the query running on the UDP returns them as local time, where the data in the UDW is shifted to UTC. I can't seem to get anything run on the UDP to match without changing the UDW query. 

Being off a few hours shouldn't make a difference to the functionality of MyLA for these columns and I think we can commit this as is and investigate that in the future if we needed to.

**Review by jonespm — APPROVED** [review]

**Pull request merged**